File: Leetcode_POTD_2025/8_August/2264_Largest_3_Same_Digit_Number_in_String.py
class Solution(object):
    def largestGoodInteger(self, num):
        possible_goods = ["999", "888", "777", "666", "555", "444", "333", "222", "111", "000"]
        for candidate in possible_goods:
            if candidate in num:
                return candidate
        return ""

File: Leetcode_POTD_2025/8_August/2264_Largest_3_Same_Digit_Number_in_String.cpp
class Solution {
public:
    string largestGoodInteger(string num) {
        vector<string> possible_goods = {"999", "888", "777", "666", "555", "444", "333", "222", "111", "000"};
        for (const string& candidate : possible_goods) {
            if (num.find(candidate) != string::npos) {
                return candidate;
            }
        }
        return "";
    }
};
